articles/atom-vs-visual-studio-code-functionality-and-features-comparison/index.md

```diff
@@ -0,0 +1,178 @@
+### Introduction
+
+This article compares the Atom and Visual Studio Code editors based on features and comparison. There are loads of code editors, but we will review Atom and Visual Studio Code editors in this article. These source code editors tend to increase the developer's productivity by providing different plugins and components. There are various features such as code-snippets, syntax highlighting, preview options, compiling, and debugging.
+
+Comparing these two text editors will enable developers to choose the editor they would love to use as per their requirement.
+
+### Table of contents
+
+- [Atom](#atom)
+
+- [Visual Studio Code Editor](#visual-studio-code-editor)
+
+- [Atom vs. VS Code: Tabular Comparison](#atom-vs-VS-code:-tabular-comparison)
+
+- [Atom vs. VS Code: Features and Functionality Comparison](#atom-vs-VS-code:-features-and-functionality-comparison)
+
+### Atom
+
+[Atom](https://atom.io/) is a free and open-source text editor developed by GitHub. It works across Windows, Linux, and Mac OS. Currently, the Atom is at version 1.52.0. It lets you easily customize every aspect of it to speed up the workflows. Atom uses a framework called Electron. Electron is a Node. js based framework that allows cross-platform in an application setting.
+
+Atom is built using web technology: HTML, CSS, and JavaScript, thus being very flexible and hackable. If you know those technologies, extending Atom is very easy. Atom features are built-in packages.
+
+Developers build Atom code/text editor with packages installed or written. There are [over 5,000 packages to choose from](https://atom.io/packages) and over 3,000 themes.
+
+Atom allows developers to use it productively without using a configuration file. Atom comes with eight syntax themes after installation. If none of the pre-installed themes amuses you, it is possible to install [customized themes from Atom](https://atom.io/themes).
+
+Developers create interactive and responsive web applications with the Atom source code editor. Learn more about Atom [here](https://atom.io/docs)
+
+#### How to Use Atom Text/Code Editor
+
+To use the Atom Code editor, you first install and download from [the Atom home page](https://atom.io/).
+
+![Atom download](/engineering-education/atom-vs-visual-studio-code-functionality-and-features-comparison/atomdownload.PNG)
+
+After clicking the download button, the Atom.zip file should appear in your downloads. Extract or Unzip the Atom file. Double click the Atom setup and follow the prompts in the Windows, Mac OS, and Linux installer to get started.
+
+Once Atom is installed, it is crucial to create an organized folder system. Files get well organized. It is easier to figure out what a particular file contains and where it goes.
```

Comment:
> Files get well organized. 

What do you mean by this? It's a little ambiguous.
Does Atom automatically organize files? And if it does, how does it handle it? Please illustrate with an example